src/expandableCalendar/Context/Provider.tsx

```diff
@@ -44,6 +44,8 @@ export interface CalendarContextProviderProps extends ViewProps {
   disabledOpacity?: number;
   /** The number of days to present in the timeline calendar */
   numberOfDays?: number;
+  /** The left inset of the timeline calendar (sidebar width), default is 72 */
+  leftInset?: number;
```

Comment:
Does it make sense to call this prop `timelineLeftInset` or `timelineSidebarWidth` - this way the prop name also hints it's only relevant to Timeline. `leftInset` feels to general

Reply:
Changed to `timelineLeftInset`